version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "1.3.4-SNAPSHOT"
+version in ThisBuild := "1.3.5-SNAPSHOT"
```

Comment:
This not necessary :smiley: The release process updates this file during a release.